ci/docker/core.build.Dockerfile

```diff
@@ -15,4 +15,5 @@ RUN pip install -U --ignore-installed  \
   -c python/requirements_compiled.txt \
   -r python/requirements.txt \
   -r python/requirements/test-requirements.txt \
-  -r python/requirements/ml/dl-cpu-requirements.txt
+  -r python/requirements/ml/dl-cpu-requirements.txt \
+  python/requirements/nsight_mock
```

Comment:
if you still need to install this, maybe install in a separate line (as it does not really have any dependencies.)